Automattic/wp-calypso
# Use white login visuals for a few oAuth flows

#### Proposed Changes

Continuing from https://github.com/Automattic/wp-calypso/pull/70334, updating few oAuth flows to be white instead of legacy visuals:

Still work in progress, current state:

**Before**

![after](https://user-images.githubusercontent.com/87168/203550563-41a3550a-b45a-419a-8e30-e4b5554a8103.png)

**After**

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/87168/203571919-b2157092-836f-4447-891d-2fc91c31dca0.png">


#### Testing Instructions


Changes should be applied only at Gravatar & Akismet pages:

* **Akismet**: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=973&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D973%26response_type%3Dcode%26blog_id%3D0%26state%3D4a9b8708620f66f5e004f26f53a9678dab2d731a4a03f3bbb962415a3b0501cf%26redirect_uri%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F%253Faction%253Drequest_access_token%26implicit%3Dtrue%26new-user%3D0)
* **Gravatar**: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D500e79a3faed390d3ef5f2778ef39363706cc0609c02c3c25397c7499f3fa51c%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token)
* Regular login: [calypso localhost link](http://calypso.localhost:3000/log-in)
* Jetpack Cloud: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0)
* Jetpack connect: [calypso localhost link](http://calypso.localhost:3000/log-in/jetpack?site=https%3A%2F%2Ffalse-shrike.jurassic.ninja&redirect_to=http%3A%2F%2Fcalypso.localhost%3A3000%2Fjetpack%2Fconnect%2Fauthorize%3Fclient_id%3D212713831%26redirect_uri%3Dhttps%253A%252F%252Ffalse-shrike.jurassic.ninja%252Fwp-admin%252Fadmin.php%253Fhandler%253Djetpack-connection-webhooks%2526action%253Dauthorize%2526_wpnonce%253D0e6a75dad7%2526redirect%253Dhttps%25253A%25252F%25252Ffalse-shrike.jurassic.ninja%25252Fwp-admin%25252Fadmin.php%25253Fpage%25253Djetpack%26state%3D1%26scope%3Dadministrator%253A3a6e1fb4a0d3fca03c4b16915582698f%26user_email%3Ddemo%2540example.org%26user_login%3Ddemo%26jp_version%3D11.5.1%26secret%3DvR0f4dhOxJWjtiika7pL3ns8z8JOsPHS%26blogname%3DFalse%2BShrike%26site_url%3Dhttps%253A%252F%252Ffalse-shrike.jurassic.ninja%26home_url%3Dhttps%253A%252F%252Ffalse-shrike.jurassic.ninja%26site_icon%26site_lang%3Den_US%26site_created%3D2022-11-23%2B12%253A55%253A29%26allow_site_connection%3D1%26locale%3Den%26_ui%3D335273%26_ut%3Dwpcom%253Auser_id%26_as%3Dwp-cli%26purchase_nonce%3DslDQZ1DY%26_wp_nonce%3D80f05dcb20%26redirect_after_auth%3Dhttps%253A%252F%252Ffalse-shrike.jurassic.ninja%252Fwp-admin%252Fadmin.php%253Fpage%253Djetpack%26site%3Dhttps%253A%252F%252Ffalse-shrike.jurassic.ninja&email_address=demo@example.org&from=%5Bunknown%5D&allow_site_connection=1)
* Crowdsignal: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D51d5e1b0d7358dee3de65c45f4b4dc37a3d6e37dab8d6047545f636c2659d104%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1)
* Happy Tools: [calypso localhost link](https://wordpress.com/log-in?client_id=46367&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D46367%26redirect_uri%3Dhttps%253A%252F%252Fschedule.happy.tools%252Fauthenticate%252Flogin%26response_type%3Dtoken%26scope%3Dauth%26state%3Dwpcom-authenticate)
* MailPoet: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=79002&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26redirect_uri%3Dhttps%253A%252F%252Faccount.mailpoet.com%252Flogin%252Fwpcom%252Fcallback%26scope%3Dauth%26client_id%3D79002)
* WooCommerce.com: [calypso localhost link](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Da982c78b99829cd0dd065f763f3cd03c4917cd6522a209fa0f42a266566442e5%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%3D%26calypso_env%3Dproduction)
* P2: [calypso localhost link](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fwordpress.com%2Fstart%2Fp2%2Fp2-confirm-email%3Fref%3Dp2-lp&from=p2&signup_url=%2Fstart%2Fp2%2Fuser%3Fref%3Dp2-lp)


#### Pre-merge Checklist
